frontend/components/AddHostsModal/PlatformWrapper/EnrollQrCode/_styles.scss

```
@@ -0,0 +1,12 @@
+.enroll-qr-code {
+  &__frame {
+    // Keeps the frame hugging the code instead of stretching to the
+    // `.form-field` column width.
+    align-self: flex-start;
+    // The SVG paints its own background and quiet zone, so the frame only
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)